dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/ApiAccessor.kt

```diff
@@ -201,60 +226,50 @@ class ApiAccessor {
         reportingPeriod: String,
         ensureQaPassed: Boolean,
     ): List<UploadInfo> {
+        fun <T> uploadCompaniesAndDatasets(
```

Comment:
Kann man die Funktion auch noch an anderen Stellen an denen "uploadCompanyAndFrameworkDataForOneFramework" verwendet wurde, verwenden? In den framework Tests wird die "uploadCompanyAndFrameworkDataForOneFramework" bisher verwendet.